firmware/quadruna/VC/src/app/vehicle_dynamics/app_pid.c

```diff
@@ -22,8 +25,20 @@ void app_pid_init(PID *pid, const PID_Config *conf)
 float app_pid_compute(PID *pid, float setpoint, float input)
 {
```

Comment:
You might b able to autogen c code from simulink u can see how they probably would handle pid.

Reply:
yeah I can look at that. I did basically use their equation for the filtering stuff tho